ui/app/AppLayouts/Communities/panels/SharedAddressesAccountSelector.qml

```diff
@@ -231,9 +232,15 @@ StatusListView {
                 inverted: true
             }
 
-            sorters: RoleSorter {
-                roleName: "symbol"
-            }
+            sorters: [
+                RoleSorter {
+                    roleName: "enabledNetworkBalance"
+                    sortOrder: Qt.DescendingOrder
+                },
```

Comment:
This is not intended behavior, it should be always alphabetical order. This prevents situation where 10000 of sth worth 1$ is before e.g. 100 ETH.

Reply:
Got it! Will remove

Reply:
Fixed